README.md

```diff
@@ -179,7 +179,7 @@ Waiting for sshd to host (10.0.0.167)
 
 Deletes a virtual machine from vSphere. If you supply `--purge` the machine will be removed from the Chef Server.
 
-NOTE: If the node name is different to the host name then the `-N` argument must be specified in conjunction with the `--purge` option
+NOTE: If the node name is different from the hostname then the `-N` argument must be specified in conjunction with the `--purge` option
```

Comment:
The `--purge` option default expects the node name and hostname to be the same. Use `-N` if the node name and hostname are different.

Reply:
Fixed